.npmrc

```diff
@@ -0,0 +1,6 @@
+public-hoist-pattern[]=*prettier*
+public-hoist-pattern[]=*eslint*
+public-hoist-pattern[]=!@typescript-eslint/*
+public-hoist-pattern[]=*ember-template-lint*
+public-hoist-pattern[]=*stylelint*
+strict-peer-dependencies=false
```

Comment:
strict-peers needs to be true for all libraries, can you delete this file? thanks!!!!

Reply:
done. Hoisting the lint packages is so far necessary for vscode plugins to properly work.